src/redux/reducers/samples/initialState.js

```diff
@@ -8,15 +8,9 @@ const sampleTemplate = {
   lastModified: null,
   complete: false,
   error: false,
-  fileNames: new Set(),
+  fileNames: [],
   files: {},
   metadata: {},
-  toJSON() {
-    return {
-      ...this,
-      fileNames: [...this.fileNames],
-    };
-  },
```

Comment:
This was added to convert `fileNames` which is a Set into an array for JSON. Filenames is also stored as a list in DynamoDB.